solr/CHANGES.txt

```diff
@@ -164,6 +164,9 @@ Bug Fixes
 
 * SOLR-17090: The v2 "delete alias" API no longer errantly returns a 405 status code (Jason Gerlowski)
 
+* SOLR-17120: Fix NullPointerException in UpdateLog.applyOlderUpdates that can occur if there are multiple partial
+  updates of the same document in separate requests using commitWithin. (Calvin Smith, Christine Poerschke)
+
```

Comment:
Tentatively adding for 9.5.0 here but we can move it to 9.6.0 if it's too late for the 9.5.0 release.

Reply:
+1 to target 9.5.0 - seems like a pretty safe change!